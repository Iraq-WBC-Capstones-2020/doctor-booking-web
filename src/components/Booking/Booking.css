@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
.booking {
  font-family: 'Roboto', sans-serif !important;
}

.booking h1 {
  color: #026fc2;
  font-size: 36px;
  font-weight: 500;
}

.booking img {
  width: 100%;
}
/*# sourceMappingURL=Booking.css.map */
/*# sourceMappingURL=Booking.css.map */
