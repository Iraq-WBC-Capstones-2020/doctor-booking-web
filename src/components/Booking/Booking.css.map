{
    "version": 3,
    "mappings": "AAAA,OAAO,CAAC,gFAAI;AAEZ,AAAA,QAAQ,CAAC;EACP,WAAW,EAAE,+BAA+B;CAgB7C;;AAjBD,AAEE,QAFM,CAEN,KAAK,CAAC;EACJ,aAAa,EAAE,IAAI;CACpB;;AAJH,AAKE,QALM,CAKN,EAAE,CAAC;EACD,KAAK,EAAE,OAAO;EACd,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,GAAG;CACjB;;AATH,AAWE,QAXM,CAWN,GAAG,CAAC;EACF,KAAK,EAAE,IAAI;CACZ;;AAbH,AAcE,QAdM,CAcN,WAAW,CAAA;EACT,KAAK,EAAE,IAAI;CACZ",
    "sources": [
        "Booking.scss"
    ],
    "names": [],
    "file": "Booking.css"
}