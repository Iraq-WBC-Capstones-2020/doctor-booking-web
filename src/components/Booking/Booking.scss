@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');

.booking {
  font-family: 'Roboto', sans-serif !important;

  h1 {
    color: #026fc2;
    font-size: 36px;
    font-weight: 500;
  }

  img {
    width: 100%;
  }
}
