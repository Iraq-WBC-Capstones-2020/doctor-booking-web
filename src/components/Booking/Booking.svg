<svg width="944" height="669" viewBox="0 0 944 669" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M944 0H40.229V536.173H944V0Z" fill="url(#paint0_linear)"/>
<path d="M933.832 63.3472H48.137V519.956H933.832V63.3472Z" fill="white"/>
<path d="M933.832 11.1489H48.137V63.3472H933.832V11.1489Z" fill="#00B0FF"/>
<path d="M923.665 96.288H60.564V494.617H923.665V96.288Z" fill="url(#paint1_linear)"/>
<path opacity="0.5" d="M175.23 103.383H71.2961V177.88H175.23V103.383Z" fill="#00B0FF"/>
<path d="M280.293 103.383H176.359V177.88H280.293V103.383Z" fill="white"/>
<path opacity="0.2" d="M385.356 103.383H281.423V177.88H385.356V103.383Z" fill="#00B0FF"/>
<path d="M490.42 103.383H386.486V177.88H490.42V103.383Z" fill="white"/>
<path d="M595.484 103.383H491.55V177.88H595.484V103.383Z" fill="white"/>
<path opacity="0.5" d="M700.547 103.383H596.613V177.88H700.547V103.383Z" fill="#00B0FF"/>
<path d="M805.61 103.383H701.676V177.88H805.61V103.383Z" fill="white"/>
<path opacity="0.2" d="M910.673 103.383H806.74V177.88H910.673V103.383Z" fill="#00B0FF"/>
<path opacity="0.2" d="M175.23 179.4H71.2961V253.896H175.23V179.4Z" fill="#00B0FF"/>
<path opacity="0.2" d="M280.293 179.4H176.359V253.896H280.293V179.4Z" fill="#00B0FF"/>
<path d="M385.356 179.4H281.423V253.896H385.356V179.4Z" fill="white"/>
<path opacity="0.5" d="M490.42 179.4H386.486V253.896H490.42V179.4Z" fill="#00B0FF"/>
<path d="M595.484 179.4H491.55V253.896H595.484V179.4Z" fill="white"/>
<path opacity="0.2" d="M700.547 179.4H596.613V253.896H700.547V179.4Z" fill="#00B0FF"/>
<path opacity="0.5" d="M805.61 179.4H701.676V253.896H805.61V179.4Z" fill="#00B0FF"/>
<path d="M910.673 179.4H806.74V253.896H910.673V179.4Z" fill="white"/>
<path d="M175.23 254.404H71.2961V328.9H175.23V254.404Z" fill="white"/>
<path d="M280.293 254.404H176.359V328.9H280.293V254.404Z" fill="white"/>
<path opacity="0.2" d="M385.356 254.404H281.423V328.9H385.356V254.404Z" fill="#00B0FF"/>
<path d="M490.42 254.404H386.486V328.9H490.42V254.404Z" fill="white"/>
<path opacity="0.2" d="M595.484 254.404H491.55V328.9H595.484V254.404Z" fill="#00B0FF"/>
<path d="M700.547 254.404H596.613V328.9H700.547V254.404Z" fill="white"/>
<path d="M805.61 254.404H701.676V328.9H805.61V254.404Z" fill="white"/>
<path opacity="0.2" d="M910.673 254.404H806.74V328.9H910.673V254.404Z" fill="#00B0FF"/>
<path d="M175.23 329.407H71.2961V403.903H175.23V329.407Z" fill="white"/>
<path opacity="0.5" d="M280.293 329.407H176.359V403.903H280.293V329.407Z" fill="#00B0FF"/>
<path opacity="0.5" d="M385.356 329.407H281.423V403.903H385.356V329.407Z" fill="#00B0FF"/>
<path d="M490.42 329.407H386.486V403.903H490.42V329.407Z" fill="white"/>
<path d="M595.484 329.407H491.55V403.903H595.484V329.407Z" fill="white"/>
<path opacity="0.5" d="M700.547 329.407H596.613V403.903H700.547V329.407Z" fill="#00B0FF"/>
<path d="M805.61 329.407H701.676V403.903H805.61V329.407Z" fill="white"/>
<path opacity="0.5" d="M910.673 329.407H806.74V403.903H910.673V329.407Z" fill="#00B0FF"/>
<path opacity="0.5" d="M175.23 405.424H71.2961V479.92H175.23V405.424Z" fill="#00B0FF"/>
<path opacity="0.2" d="M280.293 405.424H176.359V479.92H280.293V405.424Z" fill="#00B0FF"/>
<path d="M385.356 405.424H281.423V479.92H385.356V405.424Z" fill="white"/>
<path opacity="0.5" d="M490.42 405.424H386.486V479.92H490.42V405.424Z" fill="#00B0FF"/>
<path opacity="0.2" d="M595.484 405.424H491.55V479.92H595.484V405.424Z" fill="#00B0FF"/>
<path d="M700.547 405.424H596.613V479.92H700.547V405.424Z" fill="white"/>
<path opacity="0.2" d="M805.61 405.424H701.676V479.92H805.61V405.424Z" fill="#00B0FF"/>
<path d="M910.673 405.424H806.74V479.92H910.673V405.424Z" fill="white"/>
<path d="M111.401 261.498H79.769V289.878H111.401V261.498Z" fill="#69F0AE"/>
<path d="M425.461 111.492H393.829V139.872H425.461V111.492Z" fill="#69F0AE"/>
<path d="M422.072 261.498H390.44V289.878H422.072V261.498Z" fill="#69F0AE"/>
<path d="M529.395 333.461H497.763V361.841H529.395V333.461Z" fill="#69F0AE"/>
<path d="M738.392 261.498H706.76V289.878H738.392V261.498Z" fill="#69F0AE"/>
<path d="M193.802 669C300.836 669 387.605 591.153 387.605 495.124C387.605 399.094 300.836 321.247 193.802 321.247C86.7683 321.247 0 399.094 0 495.124C0 591.153 86.7683 669 193.802 669Z" fill="url(#paint2_linear)"/>
<path d="M193.803 664.307C297.948 664.307 382.375 588.561 382.375 495.124C382.375 401.686 297.948 325.94 193.803 325.94C89.6574 325.94 5.23096 401.686 5.23096 495.124C5.23096 588.561 89.6574 664.307 193.803 664.307Z" fill="#00B0FF"/>
<path d="M193.802 632.583C278.419 632.583 347.014 571.041 347.014 495.124C347.014 419.208 278.419 357.665 193.802 357.665C109.186 357.665 40.5903 419.208 40.5903 495.124C40.5903 571.041 109.186 632.583 193.802 632.583Z" fill="white"/>
<path d="M193.803 503.343C198.863 503.343 202.965 499.663 202.965 495.123C202.965 490.584 198.863 486.904 193.803 486.904C188.743 486.904 184.641 490.584 184.641 495.123C184.641 499.663 188.743 503.343 193.803 503.343Z" fill="url(#paint3_linear)"/>
<path d="M197.259 496.138L191.611 494.435L242.448 387.687L248.096 390.069L197.259 496.138Z" fill="url(#paint4_linear)"/>
<path d="M243.052 388.668L193.292 494.405L196.914 495.777L246.674 390.04L243.052 388.668Z" fill="#2196F3"/>
<path d="M225.502 551.883L192.74 497.151L197.259 495.124L231.151 548.843L225.502 551.883Z" fill="url(#paint5_linear)"/>
<path d="M196.822 495.491L193.367 497.175L226.104 551.223L229.558 549.539L196.822 495.491Z" fill="#2196F3"/>
<path d="M336.542 502.178C338.713 502.178 340.473 500.599 340.473 498.651C340.473 496.703 338.713 495.124 336.542 495.124C334.37 495.124 332.61 496.703 332.61 498.651C332.61 500.599 334.37 502.178 336.542 502.178Z" fill="#00B0FF"/>
<path d="M75.2738 434.084C76.6839 431.893 75.8471 429.091 73.4047 427.826C70.9624 426.561 67.8394 427.312 66.4294 429.503C65.0193 431.694 65.8561 434.496 68.2984 435.761C70.7407 437.026 73.8637 436.275 75.2738 434.084Z" fill="url(#paint6_linear)"/>
<path d="M74.7114 566.166C77.1537 564.901 77.9904 562.099 76.5804 559.908C75.1703 557.717 72.0474 556.966 69.6051 558.231C67.1627 559.497 66.3259 562.298 67.736 564.49C69.1461 566.681 72.269 567.432 74.7114 566.166Z" fill="url(#paint7_linear)"/>
<path d="M51.0746 503.232C53.8947 503.232 56.1809 501.181 56.1809 498.651C56.1809 496.121 53.8947 494.07 51.0746 494.07C48.2544 494.07 45.9683 496.121 45.9683 498.651C45.9683 501.181 48.2544 503.232 51.0746 503.232Z" fill="url(#paint8_linear)"/>
<path d="M125.515 388.768C127.957 387.503 128.794 384.701 127.384 382.51C125.974 380.318 122.851 379.568 120.408 380.833C117.966 382.098 117.129 384.9 118.539 387.091C119.949 389.282 123.072 390.033 125.515 388.768Z" fill="url(#paint9_linear)"/>
<path d="M196.228 372.326C198.671 371.061 199.508 368.259 198.098 366.068C196.687 363.877 193.564 363.126 191.122 364.391C188.68 365.656 187.843 368.458 189.253 370.649C190.663 372.84 193.786 373.591 196.228 372.326Z" fill="url(#paint10_linear)"/>
<path d="M266.941 388.773C269.383 387.507 270.22 384.706 268.81 382.514C267.4 380.323 264.277 379.572 261.835 380.838C259.392 382.103 258.555 384.904 259.966 387.096C261.376 389.287 264.499 390.038 266.941 388.773Z" fill="url(#paint11_linear)"/>
<path d="M319.319 435.769C321.761 434.504 322.598 431.702 321.188 429.511C319.778 427.32 316.655 426.569 314.213 427.834C311.77 429.099 310.934 431.901 312.344 434.092C313.754 436.284 316.877 437.034 319.319 435.769Z" fill="url(#paint12_linear)"/>
<path d="M318.015 566.176C320.457 564.911 321.294 562.109 319.884 559.918C318.474 557.727 315.351 556.976 312.908 558.241C310.466 559.506 309.629 562.308 311.039 564.499C312.449 566.69 315.572 567.441 318.015 566.176Z" fill="url(#paint13_linear)"/>
<path d="M264.321 611.993C266.763 610.728 267.6 607.926 266.19 605.735C264.78 603.543 261.657 602.793 259.215 604.058C256.772 605.323 255.936 608.125 257.346 610.316C258.756 612.507 261.879 613.258 264.321 611.993Z" fill="url(#paint14_linear)"/>
<path d="M196.223 626.098C198.665 624.833 199.502 622.031 198.092 619.84C196.682 617.648 193.559 616.898 191.117 618.163C188.674 619.428 187.838 622.23 189.248 624.421C190.658 626.612 193.781 627.363 196.223 626.098Z" fill="url(#paint15_linear)"/>
<path d="M128.134 611.998C130.577 610.733 131.413 607.931 130.003 605.74C128.593 603.548 125.47 602.798 123.028 604.063C120.586 605.328 119.749 608.13 121.159 610.321C122.569 612.512 125.692 613.263 128.134 611.998Z" fill="url(#paint16_linear)"/>
<path d="M193.802 371.763C195.973 371.763 197.734 370.184 197.734 368.236C197.734 366.288 195.973 364.709 193.802 364.709C191.631 364.709 189.871 366.288 189.871 368.236C189.871 370.184 191.631 371.763 193.802 371.763Z" fill="#69F0AE"/>
<path d="M267.922 386.996C269.007 385.309 268.363 383.152 266.483 382.178C264.602 381.204 262.198 381.782 261.112 383.469C260.027 385.156 260.671 387.313 262.551 388.287C264.432 389.261 266.836 388.683 267.922 386.996Z" fill="#69F0AE"/>
<path d="M318.243 434.736C320.123 433.762 320.767 431.605 319.682 429.917C318.596 428.23 316.192 427.652 314.311 428.626C312.431 429.6 311.787 431.758 312.872 433.445C313.958 435.132 316.362 435.71 318.243 434.736Z" fill="#69F0AE"/>
<path d="M318.863 563.965C319.949 562.278 319.304 560.121 317.424 559.147C315.544 558.173 313.139 558.751 312.054 560.438C310.968 562.125 311.612 564.282 313.493 565.256C315.373 566.23 317.777 565.652 318.863 563.965Z" fill="#69F0AE"/>
<path d="M263.733 610.838C265.614 609.864 266.258 607.707 265.172 606.02C264.087 604.333 261.682 603.755 259.802 604.729C257.922 605.703 257.277 607.86 258.363 609.547C259.449 611.234 261.853 611.812 263.733 610.838Z" fill="#69F0AE"/>
<path d="M125.052 388.298C126.932 387.324 127.577 385.167 126.491 383.48C125.405 381.793 123.001 381.215 121.12 382.189C119.24 383.163 118.596 385.32 119.682 387.007C120.767 388.694 123.172 389.272 125.052 388.298Z" fill="#69F0AE"/>
<path d="M74.7362 433.447C75.8218 431.76 75.1776 429.603 73.2972 428.629C71.4169 427.655 69.0125 428.233 67.9269 429.92C66.8413 431.607 67.4855 433.764 69.3659 434.738C71.2462 435.712 73.6506 435.134 74.7362 433.447Z" fill="#69F0AE"/>
<path d="M338.97 502.738C341.412 501.473 342.249 498.671 340.839 496.48C339.429 494.289 336.306 493.538 333.864 494.803C331.422 496.068 330.585 498.87 331.995 501.061C333.405 503.253 336.528 504.003 338.97 502.738Z" fill="#69F0AE"/>
<path d="M74.1237 565.254C76.0041 564.28 76.6483 562.122 75.5627 560.435C74.4771 558.748 72.0727 558.17 70.1923 559.144C68.3119 560.118 67.6677 562.276 68.7533 563.963C69.8389 565.65 72.2434 566.228 74.1237 565.254Z" fill="#69F0AE"/>
<path d="M129.241 609.552C130.327 607.865 129.683 605.707 127.802 604.733C125.922 603.759 123.518 604.338 122.432 606.025C121.346 607.712 121.991 609.869 123.871 610.843C125.751 611.817 128.156 611.239 129.241 609.552Z" fill="#69F0AE"/>
<path d="M51.0745 502.178C53.2457 502.178 55.0059 500.599 55.0059 498.651C55.0059 496.703 53.2457 495.124 51.0745 495.124C48.9032 495.124 47.1431 496.703 47.1431 498.651C47.1431 500.599 48.9032 502.178 51.0745 502.178Z" fill="#69F0AE"/>
<path d="M193.802 625.529C195.973 625.529 197.734 623.949 197.734 622.001C197.734 620.053 195.973 618.474 193.802 618.474C191.631 618.474 189.871 620.053 189.871 622.001C189.871 623.949 191.631 625.529 193.802 625.529Z" fill="#69F0AE"/>
<path opacity="0.2" d="M48.4534 508.047C48.4635 477.756 59.6234 448.317 80.1992 424.302C100.775 400.287 129.615 383.041 162.238 375.242C194.862 367.444 229.443 369.531 260.609 381.178C291.775 392.825 317.781 413.381 334.587 439.652C326.34 422.464 314.27 406.969 299.114 394.113C283.959 381.258 266.038 371.314 246.449 364.89C226.86 358.465 206.016 355.696 185.193 356.753C164.37 357.809 144.006 362.667 125.349 371.031C106.692 379.395 90.1341 391.088 76.6901 405.394C63.2461 419.7 53.1986 436.317 47.1631 454.229C41.1276 472.14 39.2311 490.968 41.5897 509.56C43.9484 528.152 50.5126 546.117 60.8803 562.353C52.656 545.2 48.4271 526.719 48.4534 508.047Z" fill="#00B0FF"/>
<path d="M193.802 501.002C197.421 501.002 200.355 498.37 200.355 495.124C200.355 491.877 197.421 489.245 193.802 489.245C190.184 489.245 187.25 491.877 187.25 495.124C187.25 498.37 190.184 501.002 193.802 501.002Z" fill="#69F0AE"/>
<defs>
<linearGradient id="paint0_linear" x1="492.114" y1="536.173" x2="492.114" y2="0" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint1_linear" x1="376036" y1="194481" x2="376036" y2="37937.6" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint2_linear" x1="66493.6" y1="229855" x2="66493.6" y2="110541" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint3_linear" x1="3328.12" y1="8651.07" x2="3328.12" y2="8384.25" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint4_linear" x1="11184.3" y1="53474.4" x2="11184.3" y2="41870.1" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint5_linear" x1="7398.89" y1="31400.5" x2="7398.89" y2="28222" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint6_linear" x1="-4140.17" y1="7689.03" x2="-4064.86" y2="7640.58" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint7_linear" x1="2580.06" y1="6780.55" x2="2580.06" y2="6697.81" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint8_linear" x1="-799.221" y1="7842.11" x2="-760.159" y2="7766.7" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint9_linear" x1="-6976.43" y1="5920.18" x2="-6884.11" y2="5920.18" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint10_linear" x1="-6318.94" y1="5834.96" x2="-6226.62" y2="5834.96" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint11_linear" x1="-5697.92" y1="6047.03" x2="-5605.7" y2="6047.03" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint12_linear" x1="-5276.83" y1="6518.92" x2="-5184.51" y2="6518.92" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint13_linear" x1="-5434.06" y1="7696.62" x2="-5341.73" y2="7696.62" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint14_linear" x1="-5970.38" y1="8062.75" x2="-5878.16" y2="8062.75" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint15_linear" x1="-6601.76" y1="8129.08" x2="-6509.43" y2="8129.08" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
<linearGradient id="paint16_linear" x1="-7201.56" y1="7940.5" x2="-7109.34" y2="7940.5" gradientUnits="userSpaceOnUse">
<stop stop-color="#808080" stop-opacity="0.25"/>
<stop offset="0.54" stop-color="#808080" stop-opacity="0.12"/>
<stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
</linearGradient>
</defs>
</svg>
