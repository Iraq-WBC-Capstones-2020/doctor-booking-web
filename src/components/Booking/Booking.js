import React, { useState } from 'react';
import './Booking.css';
import { Form, Button } from 'react-bootstrap';
import booking from './Booking.svg';
import Flatpickr from 'react-flatpickr';
import { firebaseFunctions } from '../../firebaseFunctions';

function Booking() {
  const [name, setName] = useState('');
  const [age, setAge] = useState('');
  const [gender, setGender] = useState('');
  const [phoneNumber, setPhoneNumber] = useState('');
  const [email, setEmail] = useState('');
  const [date, setDate] = useState('');
  const [time, setTime] = useState('');

  const formatDate = (date) => {
    const dateArray = date.split(' ');
    return (
      dateArray[0] +
      '-' +
      dateArray[1] +
      '-' +
      dateArray[2] +
      '-' +
      dateArray[3]
    );
  };

  const handleChange = (setField, e) => {
    setField(e.target.value);
  };

  const handleSubmit = (e) => {
    e.preventDefault();
    const appointmentInfo = {
      name: name,
      gender: gender,
      phoneNumber: phoneNumber,
      age: age,
      email: email,
      date: date,
      time: time,
      doctorUid: 'mU4KgBDdzFYjvKUhSxppqiCyFb02', //this should be changed to the uid that is passed by the link from search page
    };

    firebaseFunctions.bookAppointment(appointmentInfo);
  };

  return (
    <div className="container booking my-5">
      <h1 className="text-center">Set up an appointment</h1>
      <div className="row my-5 align-items-center">
        <div className="col-md-10 col-lg-6 mx-4 mx-sm-0">
          <Form onSubmit={handleSubmit}>
            <Form.Group controlId="name">
              <Form.Control
                type="text"
                id="name"
                placeholder="Enter your name"
                onChange={(e) => handleChange(setName, e)}
              />
            </Form.Group>

            <div className="form-group form-inline ">
              <div className="custom-control custom-radio mr-4">
                <input
                  type="radio"
                  id="female"
                  name="gender"
                  className="custom-control-input"
                  value="female"
                  onChange={(e) => handleChange(setGender, e)}
                />
                <label className="custom-control-label" htmlFor="female">
                  Female
                </label>
              </div>
              <div className="custom-control custom-radio">
                <input
                  type="radio"
                  id="male"
                  name="gender"
                  className="custom-control-input"
                  value="male"
                  onChange={(e) => handleChange(setGender, e)}
                />
                <label className="custom-control-label" htmlFor="male">
                  Male
                </label>
              </div>
            </div>

            <Form.Group controlId="age">
              <Form.Control
                type="number"
                id="age"
                placeholder="Enter your age"
                onChange={(e) => handleChange(setAge, e)}
              />
            </Form.Group>

            <Form.Group controlId="phoneNumber">
              <Form.Control
                type="number"
                id="phoneNumber"
                placeholder="Enter your phone number"
                onChange={(e) => handleChange(setPhoneNumber, e)}
              />
            </Form.Group>

            <Form.Group controlId="email">
              <Form.Control
                type="email"
                id="email"
                placeholder="Enter your  email address"
                onChange={(e) => handleChange(setEmail, e)}
              />
            </Form.Group>

            <Flatpickr
              className="time-input"
              placeholder="Choose a time"
              onChange={(date) => {
                setDate(formatDate(date.toString()));
              }}
            />

            <Flatpickr
              className="time-input"
              placeholder="Choose a time"
              options={{
                noCalendar: true,
                enableTime: true,
              }}
              onChange={(time) => {
                setTime(time.toString().split(' ')[4]);
              }}
            />

            <Button variant="primary" type="submit">
              Set Appoinment
            </Button>
          </Form>
        </div>
        <div className=" d-none d-lg-block col-lg-6 ">
          <img src={booking} alt="" />
        </div>
      </div>
    </div>
  );
}

export default Booking;
