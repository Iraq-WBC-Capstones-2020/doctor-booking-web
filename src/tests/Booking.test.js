import React from 'react';
import Booking from '../components/Booking/Booking';
import { render } from '@testing-library/react';

test('Boooking page renders', () => {
  render(<Booking />);
});
// test('Boooking page run well run well ', () => {
//   const tree = renderer.create(<Booking />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
