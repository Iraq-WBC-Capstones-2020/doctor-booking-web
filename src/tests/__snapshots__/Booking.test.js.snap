// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Boooking page run well run well  1`] = `
<div
  className="container booking my-5"
>
  <h1
    className="text-center"
  >
    Set up an appointment
  </h1>
  <div
    className="row my-5 align-items-center"
  >
    <div
      className="col-md-10 col-lg-6 mx-4 mx-sm-0"
    >
      <form
        className=""
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <input
            className="form-control"
            id="name"
            onChange={[Function]}
            placeholder="Enter your name"
            type="text"
          />
        </div>
        <div
          className="form-group form-inline "
        >
          <div
            className="custom-control custom-radio mr-4"
          >
            <input
              className="custom-control-input"
              id="female"
              name="gender"
              onChange={[Function]}
              type="radio"
              value="female"
            />
            <label
              className="custom-control-label"
              htmlFor="female"
            >
              Female
            </label>
          </div>
          <div
            className="custom-control custom-radio"
          >
            <input
              className="custom-control-input"
              id="male"
              name="gender"
              onChange={[Function]}
              type="radio"
              value="male"
            />
            <label
              className="custom-control-label"
              htmlFor="male"
            >
              Male
            </label>
          </div>
        </div>
        <div
          className="form-group"
        >
          <input
            className="form-control"
            id="age"
            onChange={[Function]}
            placeholder="Enter your age"
            type="number"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="form-control"
            id="phoneNumber"
            onChange={[Function]}
            placeholder="Enter your phone number"
            type="number"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="form-control"
            id="email"
            onChange={[Function]}
            placeholder="Enter your  email address"
            type="email"
          />
        </div>
        <button
          className="btn btn-primary"
          disabled={false}
          type="submit"
        >
          Set Appoinment
        </button>
      </form>
    </div>
    <div
      className=" d-none d-lg-block col-lg-6 "
    >
      <img
        alt=""
        src="Booking.svg"
      />
    </div>
  </div>
</div>
`;
